<!doctype html>
<html lang="en">
<head>
  <meta charset="utf-8">
  <title>Quantitative Genetics</title>
  <meta name="description" content="">
  <meta name="author" content="Antonio Augusto F Garcia">
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black-translucent" />
  <link rel="stylesheet" href="http://netdna.bootstrapcdn.com/bootswatch/2.3.1/amelia/bootstrap.min.css">
<link rel="stylesheet" href="./frameworks/revealjs/css/reveal.min.css">
  <link rel="stylesheet" href="./frameworks/revealjs/css/theme/night.css" id="theme">
  <link rel="stylesheet" href="./highlighters/highlight.js/css/zenburn.css" id="theme">
  <!--[if lt IE 9]>
  <script src="lib/js/html5shiv.js"></script>
  <![endif]-->  <link rel="stylesheet" href = "assets/css/slidify-reveal.css">

</head>
<body>
  
  <div class="reveal">
    <div class="slides">
      <section class='' data-state='' id='slide-1'>
  <h1>Quantitative Genetics</h1>
  <h2>Decomposition of The Genetic Variance</h2>

<p><small>
Instructor: <a href="http://about.me/augusto.garcia">A Augusto F Garcia</a>
/ <a href="http://statgen.esalq.usp.br">Statistical Genetics Lab</a> </small></p>

<p><small>Department of Genetics, Luiz de Queiroz College of Agriculture,
University of SÃ£o Paulo (Brazil)</small></p>

<script src="http://ajax.googleapis.com/ajax/libs/jquery/1.9.1/jquery.min.js"></script>

  <aside class='notes'>
    <p>Some notes on the first slide</p>

  </aside>
</section>
<section class='' data-state='' id='slide-2'>
  <h1>Content</h1>
  <h3>Influence of a locus on the phenotype</h3>

<h3>Fisher&#39;s decomposition of the genotypic value</h3>

<h3>Partitioning the genetic variance</h3>

<h3>Additive effects, average excesses, breeding values</h3>

<h3>Extensions for multiple alleles</h3>

</section>
<section class='' data-state='' id='slide-3'>
  <h3>Influence of a locus on the phenotype</h3>
  <p>\[z = G + E\]</p>

<ul>
<li>\(G\) is for the whole genotype (which is very complicated)</li>
<li>Let us start with a single gene
<img src="./images/Figure4-4.jpg" alt=""></li>
<li>\(k\) is the dominance

<ul>
<li>\(k=0\) (no dominance), \(k=1\) (full dominance), \(k\geq 1\)
(overdominance)</li>
</ul></li>
</ul>

</section>
<section class='' data-state='' id='slide-4'>
  <h1>Fisher&#39;s decomposition (1918)</h1>
  <ul>
<li>Number of copies of a particular allele: <strong>gene content</strong></li>
<li>Relation between gene content and genotypic value (normally,
non-linear)</li>
<li>Partitioning genotypic values into their &quot;expected&quot; values based on
additivity (\(\hat{G}\)) and deviations from those expectations</li>
<li>Deviations: dominance (\(\delta\))</li>
</ul>

</section>
<section class='' data-state='' id='slide-5'>
  <h3>Regression model</h3>
  <p>\[G_{ij}=\hat{G}_{ij}+\delta_{ij}=\mu_G+\alpha_1N_1+\alpha_2N2+\delta_{ij}\]</p>

<ul>
<li>This is a multiple regression model</li>
<li>However, there is a constraint: \(N_1=2-N_2\)
\[G_{ij}=\mu_{G}+\alpha_1(2-N_2)+\alpha_2N_2+\delta_{ij}\]
\[G_{ij}=\iota + (\alpha_2-\alpha_1)N_2 + \delta_{ij}\]</li>
<li>Where \(\iota=\mu_G+2\alpha_1\)</li>
<li>The slope of this regression can be denoted as \(\alpha=\alpha_2-\alpha_1\)</li>
<li>The (regression )model is therefore
\[G_{ij}=\iota+\alpha N_2 + \delta_{ij}\]</li>
</ul>

</section>
<section class='' data-state='' id='slide-6'>
  <h3>Graphical interpretation</h3>
  <p><img src="./images/Figure4-6.jpg" alt=""></p>

</section>
<section class='' data-state='' id='slide-7'>
  
  <ul>
<li>Notice that
\[\hat{G}_{ij}=\mu_{G}+\alpha_{i}+\alpha_{j} = \left\{
\begin{array}{lr} \mu_G+2\alpha_1 & for\; G_{11}\\
\mu_G+\alpha_1+\alpha_2 & for\; G_{21}\\
\mu_G+2\alpha_2 & for\; G_{22}\end{array}\right.\]</li>
<li>By taking the expectation of
\(G_{ij}=\mu_G+\alpha_1N_1+\alpha_2N_2+\delta_{ij}\), it can be shown
that
\[\alpha_2=p_1\alpha\]
\[\alpha_1=-p_2\alpha\]</li>
<li>\(p_1\) (\(p_2\)) is the frequency of allele \(B_1\) (\(B_2\))</li>
</ul>

</section>
<section class='' data-state='' id='slide-8'>
  
  <ul>
<li>The slope of the regression (graphical representation) is
\[\alpha=\frac{\sigma(G,N_2)}{\sigma^2(N_2)}\]</li>
<li>To obtain \(\alpha\), one needs to make a number of intermediate
calculations

<ul>
<li>\(E(N_2)\) and \(E(N_2^2)\) for \(\sigma^2(N_2)\)</li>
<li>\(E(GN)\), \(E(G)\) and \(E(N_2)\) for \(\sigma(G,N_2)\)</li>
<li>Expectations require &quot;values&quot; and &quot;frequencies&quot;</li>
</ul></li>
<li><p>Finally:</p>

<p>\[\alpha=a[1+k(p_1-p_2)]\]</p></li>
</ul>

</section>
<section class='' data-state='' id='slide-9'>
  
  <ul>
<li>Average effect of allelic substitution (under random mating)
\[\alpha=a[1+k(p_1-p_2)]\]</li>
<li>It represents the average change in genotypic value that results
when a \(B_1\) allele is randomly substituted for a \(B_2\) allele

<ul>
<li>If \(k=0\), \(\alpha=a\)</li>
<li>If \(p_1=p_2\) (for example, in an \(F_2\) population), \(\alpha=a\)</li>
<li>With dominance, the effect of an allelic substitution depends on
the status of the unsubstituted allele</li>
<li>The effect of an allele can only be defined in the context of a
population</li>
</ul></li>
</ul>

</section>
<section class='' data-state='' id='slide-10'>
  
  <p><img src="./images/Figure4-7.jpg" alt=""></p>

</section>
<section class='' data-state='indigo' id='slide-11'>
  <h1>Homework</h1>
  <ul>
<li>Read Chapter 2 and 3 of Lynch and Walsh Book</li>
</ul>

</section>
    </div>
  </div>
</body>
  <script src="./frameworks/revealjs/lib/js/head.min.js"></script>
  <script src="./frameworks/revealjs/js/reveal.min.js"></script>
  <script>
  // Full list of configuration options available here:
  // https://github.com/hakimel/reveal.js#configuration
  Reveal.initialize({
    controls: true,
    progress: true,
    history: true,
    center: true,
    theme: Reveal.getQueryHash().theme || 'night', 
    transition: Reveal.getQueryHash().transition || 'linear', 
    dependencies: [
    // Cross-browser shim that fully implements classList -
    // https://github.com/eligrey/classList.js/
      { src: './frameworks/revealjs/lib/js/classList.js', condition: function() { return !document.body.classList;}},
      // Zoom in and out with Alt+click
      { src: './frameworks/revealjs/plugin/zoom-js/zoom.js', async: true, condition: function() { return !!document.body.classList; } },
      // Speaker notes
      { src: './frameworks/revealjs/plugin/notes/notes.js', async: true, condition: function() { return !!document.body.classList; } },
      // Remote control your reveal.js presentation using a touch device
      //{ src: './frameworks/revealjs/plugin/remotes/remotes.js', async: true, condition: function() { return !!document.body.classList; } }
      ]
  });
  </script>  <!-- MathJax: Fall back to local if CDN offline but local image fonts are not supported (saves >100MB) -->
<script type="text/x-mathjax-config">
  MathJax.Hub.Config({
    tex2jax: {
      inlineMath: [['$','$'], ['\\(','\\)']],
      processEscapes: true
    }
  });
</script>
<script type="text/javascript" src="http://cdn.mathjax.org/mathjax/2.0-latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>
<!-- <script src="https://c328740.ssl.cf1.rackcdn.com/mathjax/2.0-latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML">
</script> -->
<script>window.MathJax || document.write('<script type="text/x-mathjax-config">MathJax.Hub.Config({"HTML-CSS":{imageFont:null}});<\/script><script src="./widgets/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"><\/script>')
</script>
<!-- LOAD HIGHLIGHTER JS FILES -->
<script src="./highlighters/highlight.js/highlight.pack.js"></script>
<script>hljs.initHighlightingOnLoad();</script>
<!-- DONE LOADING HIGHLIGHTER JS FILES -->
 

</html>