<!doctype html>
<html lang="en">
<head>
  <meta charset="utf-8">
  <title>Quantitative Genetics</title>
  <meta name="description" content="">
  <meta name="author" content="Antonio Augusto F Garcia">
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black-translucent" />
  <link rel="stylesheet" href="http://netdna.bootstrapcdn.com/bootswatch/2.3.1/amelia/bootstrap.min.css">
<link rel="stylesheet" href="./frameworks/revealjs/css/reveal.min.css">
  <link rel="stylesheet" href="./frameworks/revealjs/css/theme/night.css" id="theme">
  <link rel="stylesheet" href="./highlighters/highlight.js/css/zenburn.css" id="theme">
  <!--[if lt IE 9]>
  <script src="lib/js/html5shiv.js"></script>
  <![endif]-->  <link rel="stylesheet" href = "assets/css/slidify-reveal.css">

</head>
<body>
  
  <div class="reveal">
    <div class="slides">
      <section class='' data-state='' id='slide-1'>
  <h1>Quantitative Genetics</h1>
  <h2>Decomposition of The Genetic Variance</h2>

<p><small>
Instructor: <a href="http://about.me/augusto.garcia">A Augusto F Garcia</a>
/ <a href="http://statgen.esalq.usp.br">Statistical Genetics Lab</a> </small></p>

<p><small>Department of Genetics, Luiz de Queiroz College of Agriculture,
University of SÃ£o Paulo (Brazil)</small></p>

<script src="http://ajax.googleapis.com/ajax/libs/jquery/1.9.1/jquery.min.js"></script>

  <aside class='notes'>
    <p>Some notes on the first slide</p>

  </aside>
</section>
<section class='' data-state='' id='slide-2'>
  <h1>Content</h1>
  <h3>Influence of a locus on the phenotype</h3>

<h3>Fisher&#39;s decomposition of the genotypic value</h3>

<h3>Partitioning the genetic variance</h3>

<h3>Additive effects and breeding values</h3>

<h3>Extensions for multiple alleles</h3>

</section>
<section class='' data-state='' id='slide-3'>
  <h3>Influence of a locus on the phenotype</h3>
  <p>\[z = G + E\]</p>

<ul>
<li>\(G\) is for the whole genotype (which is very complicated)</li>
<li>Let us start with a single gene
<img src="./images/Figure4-4.jpg" alt=""></li>
<li>\(k\) is the dominance

<ul>
<li>\(k=0\) (no dominance), \(k=1\) (full dominance), \(k\geq 1\)
(overdominance)</li>
</ul></li>
</ul>

</section>
<section class='' data-state='' id='slide-4'>
  <h1>Fisher&#39;s decomposition (1918)</h1>
  <ul>
<li>Number of copies of a particular allele: <strong>gene content</strong></li>
<li>Relation between gene content and genotypic value (normally,
non-linear)</li>
<li>Partitioning genotypic values into their &quot;expected&quot; values based on
additivity (\(\hat{G}\)) and deviations from those expectations</li>
<li>Deviations: dominance (\(\delta\))</li>
</ul>

</section>
<section class='' data-state='' id='slide-5'>
  <h3>Regression model</h3>
  <p>\[G_{ij}=\hat{G}_{ij}+\delta_{ij}=\mu_G+\alpha_1N_1+\alpha_2N_2+\delta_{ij}\]</p>

<ul>
<li>This is a multiple regression model</li>
<li>However, there is a constraint: \(N_1=2-N_2\)
\[G_{ij}=\mu_{G}+\alpha_1(2-N_2)+\alpha_2N_2+\delta_{ij}\]
\[G_{ij}=\iota + (\alpha_2-\alpha_1)N_2 + \delta_{ij}\]</li>
<li>Where \(\iota=\mu_G+2\alpha_1\)</li>
<li>The slope of this regression can be denoted as \(\alpha=\alpha_2-\alpha_1\)</li>
<li>The (regression )model is therefore
\[G_{ij}=\iota+\alpha N_2 + \delta_{ij}\]</li>
</ul>

</section>
<section class='' data-state='' id='slide-6'>
  <h3>Graphical interpretation</h3>
  <p><img src="./images/Figure4-6.jpg" alt=""></p>

</section>
<section class='' data-state='' id='slide-7'>
  
  <ul>
<li>Notice that
\[\hat{G}_{ij}=\mu_{G}+\alpha_{i}+\alpha_{j} = \left\{
\begin{array}{lr} \mu_G+2\alpha_1 & for\; G_{11}\\
\mu_G+\alpha_1+\alpha_2 & for\; G_{21}\\
\mu_G+2\alpha_2 & for\; G_{22}\end{array}\right.\]</li>
<li>By taking the expectation of
\(G_{ij}=\mu_G+\alpha_1N_1+\alpha_2N_2+\delta_{ij}\), it can be shown
that
\[\alpha_2=p_1\alpha\]
\[\alpha_1=-p_2\alpha\]</li>
<li>\(p_1\) (\(p_2\)) is the frequency of allele \(B_1\) (\(B_2\))</li>
</ul>

</section>
<section class='' data-state='' id='slide-8'>
  
  <ul>
<li>The slope of the regression (graphical representation) is
\[\alpha=\frac{\sigma(G,N_2)}{\sigma^2(N_2)}\]</li>
<li>To obtain \(\alpha\), one needs to make a number of intermediate
calculations

<ul>
<li>\(E(N_2)\) and \(E(N_2^2)\) for \(\sigma^2(N_2)\)</li>
<li>\(E(GN)\), \(E(G)\) and \(E(N_2)\) for \(\sigma(G,N_2)\)</li>
<li>Expectations require &quot;values&quot; and &quot;frequencies&quot;</li>
</ul></li>
<li><p>Finally:</p>

<p>\[\alpha=a[1+k(p_1-p_2)]\]</p></li>
</ul>

</section>
<section class='' data-state='' id='slide-9'>
  
  <ul>
<li>Average effect of allelic substitution (under random mating)
\[\alpha=a[1+k(p_1-p_2)]\]</li>
<li>It represents the average change in genotypic value that results
when a \(B_1\) allele is randomly substituted for a \(B_2\) allele

<ul>
<li>If \(k=0\), \(\alpha=a\)</li>
<li>If \(p_1=p_2\) (for example, in an \(F_2\) population), \(\alpha=a\)</li>
<li>With dominance, the effect of an allelic substitution depends on
the status of the unsubstituted allele</li>
<li>The effect of an allele can only be defined in the context of a
population</li>
</ul></li>
</ul>

</section>
<section class='' data-state='' id='slide-10'>
  
  <table><thead>
<tr>
<th align="center">Genotype</th>
<th align="center">Genotypic Value (\(G\))</th>
<th align="center">Gene content (\(N_2\))</th>
<th align="center">Freq.</th>
<th align="center">\(\hat{G}\)</th>
<th align="center">\(\delta=G-\hat{G}\)</th>
</tr>
</thead><tbody>
<tr>
<td align="center">\(B_1B_1\)</td>
<td align="center">\(0\)</td>
<td align="center">\(0\)</td>
<td align="center">\(p_1^2\)</td>
<td align="center">\(\iota\)</td>
<td align="center">\(-\iota\)</td>
</tr>
<tr>
<td align="center">\(B_1B_2\)</td>
<td align="center">\((1+k)a\)</td>
<td align="center">\(1\)</td>
<td align="center">\(2p_1p_2\)</td>
<td align="center">\(\iota+\alpha\)</td>
<td align="center">\((1+k)a-\iota-\alpha\)</td>
</tr>
<tr>
<td align="center">\(B_2B_2\)</td>
<td align="center">\(2a\)</td>
<td align="center">\(2\)</td>
<td align="center">\(p_2^2\)</td>
<td align="center">\(\iota+2\alpha\)</td>
<td align="center">\(2a-\iota-2\alpha\)</td>
</tr>
</tbody></table>

</section>
<section class='' data-state='' id='slide-11'>
  
  <p><img src="./images/Figure4-7.jpg" alt=""></p>

</section>
<section class='' data-state='' id='slide-12'>
  <h1>Partitioning the genetic variance</h1>
  <ul>
<li>Given that \(G = \hat{G} + \delta\), the total genetic variance is
\[\sigma_G^2=\sigma^2(\hat{G}+\delta)=\sigma^2(\hat{G})+2\sigma(\hat{G},\delta)+\sigma^2(\delta)\]</li>
<li>Recall from regression properties that the predictor and the
residual are uncorrelated (in this case, \(\hat{G}\) and \(\delta\))</li>
<li>So, we can denote these components as
\[\sigma^2_G=\sigma^2_A+\sigma^2_D\]

<ul>
<li>\(\sigma^2_A\) is the amount of variation explained by the
regression on \(N_2\) (<strong>additive genetic variance</strong>)</li>
<li>\(\sigma^2_D\) is the residual variance for the regression
(<strong>dominance genetic variance</strong>)</li>
</ul></li>
</ul>

</section>
<section class='' data-state='' id='slide-13'>
  <h2>Genetic variance</h2>
  <ul>
<li><p>Using \(E(\hat{G}^2)\), \(E(\hat{G})\), \(E(\delta^2)\), \(E(\delta)\), we
can obtain:</p>

<p>\[\sigma^2_A=E(\hat{G}^2)-[E(\hat{G})]^2=2p_1p_2\alpha^2\]</p>

<p>\[\sigma^2_D=E(\delta^2)-[E(\delta)]^2=(2p_1p_2ak)^2\]</p></li>
</ul>

</section>
<section class='' data-state='' id='slide-14'>
  
  <ul>
<li>solid line is \(\sigma^2_G\), dashed line is \(\sigma^2_A\), dotted line
is \(\sigma^2_D\)</li>
<li>\(\sigma^2_A/\sigma^2_D\) <strong>does not</strong> provide information about
additivity of gene action!</li>
</ul>

<p><img src="./images/Figure4-8.jpg" alt=""></p>

</section>
<section class='' data-state='' id='slide-15'>
  <h1>Additive effects and breeding values</h1>
  <ul>
<li>In randomly mating diploids, each parent donates only <strong>one allele
per locus</strong> to its offspring</li>
<li>Additive effects are for each allele</li>
<li>Dominance deviations are functions of the interaction between the
two parental alleles</li>
<li>So, dominance is eliminated when gametes are produced</li>
</ul>

<blockquote>
<p>\(\hat{G}\) and \(\delta\) are the heritable and non-heritable
components of an individual&#39;s genotypic value</p>
</blockquote>

</section>
<section class='' data-state='' id='slide-16'>
  <h3>Additive effects</h3>
  <ul>
<li>The <em>additive effects</em> \(\alpha_i\) are defined to be the
least-squares regression coefficients of genotypic values on gene
content:
\[\alpha_2=p_1\alpha=p_1a[1+k(p_1-p_2)]\]
\[\alpha_1=-p_2\alpha=-p_2a[1+k(p_1-p_2)]\]</li>
<li>They are equivalent to what is defined as <em>average excesses</em> in
random mating populations (&quot;the difference between the mean
genotypic value of individuals carrying at least one copy of a given
allele and the mean genotypic value of the entire population&quot;)</li>
</ul>

</section>
<section class='' data-state='blue' id='slide-17'>
  <h3>Breeding value</h3>
  <blockquote>
<p>An individual&#39;s <strong>breeding value</strong> (\(A\)) is the sum of the additive
effects of its genes</p>
</blockquote>

<ul>
<li>The breeding value of \(B_1B_1\) is \(2\alpha_1\)</li>
<li>The breeding value of \(B_1B_2\) is \(\alpha_1+\alpha_2\)</li>
<li>The breeding value of \(B_2B_2\) is \(2\alpha_2\)</li>
</ul>

</section>
<section class='' data-state='' id='slide-18'>
  <h4>Breeding value</h4>
  <ul>
<li>If one crosses individuals with a given breeding value with a random
sample of the entire population, it is possible to estimate the
average genotypic value of offspring. By subtracting the mean, we
have a very useful result.</li>
<li>Deviations are <em>half</em> of the breeding value </li>
</ul>

<table><thead>
<tr>
<th>Parental genotype</th>
<th>Breeding value</th>
<th>Mean genotypic value of progeny</th>
<th>Deviation of expected progeny mean from \(\mu_G\)</th>
</tr>
</thead><tbody>
<tr>
<td>\(B_2B_2\)</td>
<td>\(2\alpha_2\)</td>
<td>\(p_2(2a)+p_1a(1+k)\)</td>
<td>\(\alpha_2\)</td>
</tr>
<tr>
<td>\(B_1B_2\)</td>
<td>\(\alpha_1+\alpha_2\)</td>
<td>\(a[p_2+(1+k)/2]\)</td>
<td>\((\alpha_1+\alpha_2)/2\)</td>
</tr>
<tr>
<td>\(B_1B_1\)</td>
<td>\(2\alpha_1\)</td>
<td>\(ap_2(1+k)\)</td>
<td>\(\alpha_1\)</td>
</tr>
</tbody></table>

<blockquote>
<p>We can estimate the breeding value by crossing to random
individuals, taking the average of progeny, subtracting it from the
overall mean, and then multiplying by 2</p>
</blockquote>

</section>
<section class='' data-state='' id='slide-19'>
  <h2>A very useful interpretation</h2>
  <ul>
<li>Remember that \(G_{ij}=\mu_G+\alpha_i+\alpha_j+\delta_{ij}\)</li>
<li>This is the same as using the model
\(G_{ij}=\mu_G+\alpha_1N_1+\alpha_2N_2+\delta_{ij}\)</li>
<li>\(G_{ij}=\mu_G+A_{ij}+\delta_{ij}\) (indicating the breeding value)</li>
<li>Then
\[\sigma^2_G=\sigma^2(\alpha_i+\alpha_j)+\sigma^2(\delta_{ij})\]</li>
<li>But
\[\sigma^2_G=\sigma^2_A+\sigma^2_D\]</li>
</ul>

<blockquote>
<p>The genetic variance is equivalent to the variance of breeding
values of individuals in the population</p>
</blockquote>

</section>
<section class='' data-state='' id='slide-20'>
  <h1>Extension for multiple alleles</h1>
  <ul>
<li>Generalizations:
\[G=\mu_G+\sum_{i=1}^n \alpha_iN_i+\delta\]
\[\alpha_i=\sum_{j=1}^n p_jG_{ij}-\mu_G\]
\[\sigma^2_A=2\sum_{i=1}^n p_i\alpha^2_i\]</li>
</ul>

</section>
<section class='' data-state='indigo' id='slide-21'>
  <h1>Homework</h1>
  <ul>
<li>Read Chapter 4 of Lynch and Walsh Book</li>
</ul>

</section>
    </div>
  </div>
</body>
  <script src="./frameworks/revealjs/lib/js/head.min.js"></script>
  <script src="./frameworks/revealjs/js/reveal.min.js"></script>
  <script>
  // Full list of configuration options available here:
  // https://github.com/hakimel/reveal.js#configuration
  Reveal.initialize({
    controls: true,
    progress: true,
    history: true,
    center: true,
    theme: Reveal.getQueryHash().theme || 'night', 
    transition: Reveal.getQueryHash().transition || 'linear', 
    dependencies: [
    // Cross-browser shim that fully implements classList -
    // https://github.com/eligrey/classList.js/
      { src: './frameworks/revealjs/lib/js/classList.js', condition: function() { return !document.body.classList;}},
      // Zoom in and out with Alt+click
      { src: './frameworks/revealjs/plugin/zoom-js/zoom.js', async: true, condition: function() { return !!document.body.classList; } },
      // Speaker notes
      { src: './frameworks/revealjs/plugin/notes/notes.js', async: true, condition: function() { return !!document.body.classList; } },
      // Remote control your reveal.js presentation using a touch device
      //{ src: './frameworks/revealjs/plugin/remotes/remotes.js', async: true, condition: function() { return !!document.body.classList; } }
      ]
  });
  </script>  <!-- MathJax: Fall back to local if CDN offline but local image fonts are not supported (saves >100MB) -->
<script type="text/x-mathjax-config">
  MathJax.Hub.Config({
    tex2jax: {
      inlineMath: [['$','$'], ['\\(','\\)']],
      processEscapes: true
    }
  });
</script>
<script type="text/javascript" src="http://cdn.mathjax.org/mathjax/2.0-latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>
<!-- <script src="https://c328740.ssl.cf1.rackcdn.com/mathjax/2.0-latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML">
</script> -->
<script>window.MathJax || document.write('<script type="text/x-mathjax-config">MathJax.Hub.Config({"HTML-CSS":{imageFont:null}});<\/script><script src="./widgets/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"><\/script>')
</script>
<!-- LOAD HIGHLIGHTER JS FILES -->
<script src="./highlighters/highlight.js/highlight.pack.js"></script>
<script>hljs.initHighlightingOnLoad();</script>
<!-- DONE LOADING HIGHLIGHTER JS FILES -->
 

</html>